### DISCUSSION TITLE
Error when running “zeus dev”

### DISCUSSION THREAD

**Discussion by 2382958980** [zepp-health/zeppos-forum#245] [Q&A] [answered]
Followed the documents,  and downloaded sample app on github, but when I try to run "zeus dev", there comes the error. I have searched this problem on internet but it seems that this error doesn't occur on other's device except me. the problem might be related to"zeppos-app-utils" so I reassembly this package for some times, I also tried other ways but nothing is helpful. If anyone can solve my prblem, I will be truly grateful.
![20230924-213140](https://github.com/zepp-health/zeppos-forum/assets/59905973/9f781d9a-2c3f-43d5-a2db-d0b1c9155b89)
)

**Comment by silver-zepp** [comment, marked as answer]
Please try these steps @2382958980 

Make sure you have up-to-date main libs:
`npm install @zeppos/zpm -g`
`npm install @zeppos/zeus-cli -g`
(restart your terminal/VSCode after the update)


Then I would suggest re-installing the LTS version of Node.js (not the "Current" one):
https://nodejs.org/en/download

If that still doesn't work last suggestion would be to move the zeppos-sample-main library to the root of your `D:` drive, making sure you don't have complex Unicode characters in the path.

Let me know if that fixes the issue!

**Reply by 2382958980** [reply]
Appreciate for your solution. In fact I forgot ‘-g’ when installing @zeppos/zeus-cli. Apologize for any confusion caused by my weird-grammer descrption. Thanks again.

**Reply by Colinnnng** [reply]
Check the simulator install and set already.
cd /opt/simulator/ && ./simulator
